README.md

```diff
@@ -120,15 +121,17 @@ Hurray! You successfully made a contribution! 🎉
 
 ## What if I have a Conflict?
 
-#### A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged in order to prevent collision in the main branch.
+### A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged in order to prevent collision in the main branch
```

Comment:
```suggestion
A GitHub conflict is when people make changes to the same area or line in a file. This must be fixed before it is merged in order to prevent collision in the main branch.
```